installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc

```diff
@@ -59,9 +59,19 @@ include::modules/ipi-install-configure-network-components-to-run-on-the-control-
 
 include::modules/ipi-install-deploying-routers-on-worker-nodes.adoc[leveloffset=+2]
 
-include::modules/ipi-install-configuring-bios-for-worker-node.adoc[leveloffset=+2]
+include::modules/ipi-install-configuring-the-bios.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../post_installation_configuration/bare-metal-configuration.html[Bare metal configuration]
```

Comment:
The xref needs to use the `.adoc` extension and also needs an anchor `#<anchor>`.